README.md

```diff
@@ -110,76 +109,35 @@ npm run test:watch
+### MVP
+The [MVP user stories are shown in the MVP Project](https://github.com/freeCodeCamp/chapter/projects/1) kanban / cards and as [issues marked with "MVP"](https://github.com/freeCodeCamp/chapter/labels/MVP).
 
-### As a user
+### Post-MVP
+We are maintaining a list of [post-MVP user stories](https://github.com/freeCodeCamp/chapter/issues/84).
 
-- I can open a registration page where I can sign up with email and password.
+### Suggesting a New Story
 
-- I can log in with my email and password or I can log in with social login via Google.
-
-- I can see my account page where I can reset my password if I've registered with email and password. Otherwise I'll see a link to my Google profile.
-
-- I can log out.
-
-### As a future participant
-
-- I can use a search box on the landing page to input a city, state, or country name and it will autocomplete. I can click one of those locations.
-
-- When I click one of those locations, I can see the "show view" for that event's chapter, with details about the upcoming event, along with a button to RSVP.
-
-- I can click the "RSVP" button. When I do, I will be prompted to sign in. Then I will receive an email with a ticket and I will be added to the public list of event attendees.
-
-- I will receive a second email the day before the event to remind me.
-
-- After the event, I will automatically get emails notifying me of subsequent events.
-
-- I can filter all events in my location by tag/interests.
-
-### As an organizer
-
-- I can create a chapter.
-
-- I can edit details about the chapter, including a Slack/Discord/Facebook/WeChat/WhatsApp link participants can join to discuss and coordinate events.
-
-- I can create events, and set their location and capacity.
-
-- I can cancel events.
-
-- I can email the entire list of participants.
-
-- I can ban a participant whom I believe is toxic or who has previously broken my organization's code of conduct.
-
-- I can add a venue sponsor to the event with a link to their website as a way of thanking them for hosting.
-
-- I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
-
-- I can see how many times a participant has come to the event as well as their attendance rate.
-
-- I can check-in attendees on the event registration desk with their `email_id` or `chapter_id`.
+1. Please check your idea does not already exist as an [MVP issue](https://github.com/freeCodeCamp/chapter/labels/MVP), or in the [post-MVP list](https://github.com/freeCodeCamp/chapter/issues/84).
+1. Create a new "Issue" and post details about the suggested user story.
 
 ## Roadmap
 
-1. Design the schema.
-2. Set up the API endpoints.
-3. Build the web client and let other developers use the API to build mobile clients and voice interface clients.
+The on-going [project Roadmap conversation](https://github.com/freeCodeCamp/chapter/issues/47) is regularly updated to reflect the overall progress and for higher-level discussions.
```

Comment:
Add this below the old roadmap. Something like:

```suggestion
1. Design the schema - Done.
2. Set up the API endpoints - Done.
3. Built the web client and let other developers use the API to build mobile clients and voice interface clients.

We have a more detailed information on roadmap on the [Roadmap issue]https://github.com/freeCodeCamp/chapter/issues/47). It is regularly updated to reflect the overall progress and for higher-level discussions.
```

Reply:
If someone else is going to maintain the bullets here then I'm fine with it. As it was, this list was out of sync with #47, which has the established roadmap and is much easier / faster to edit.

Reply:
@allella Link to the issue seems right.